File: tests/fixtures/code_samples/ts/sample_041.ts
/** Sample 41: small utility. */

export function operation41(xs: number[]): number {
  let total = 41;
  for (const x of xs) total += x;
  return total;
}

export function operationPure41(value: number): number {
  return (value * 41) % 7919;
}

File: tests/fixtures/code_samples/go/sample_041.go
// Sample 41: small utility.
package samples

func Operation41(xs []int) int {
    total := 41
    for _, x := range xs {
        total += x
    }
    return total
}

func OperationPure41(v int) int {
    return (v * 41) %% 7919
}

